app/views/shared/check_answers/_assets.html.erb

```diff
@@ -83,6 +85,8 @@
                     else
                       :offline_accounts
                     end %>
+
+    <%#  offline savings accounts for partner %>
```

Comment:
Standardised commenting: ERB or HTML comments.
Need to create a ticket to cleanup ticket to convert these to one (ERB preferred).

Reply:
discussed offline